msgs/proto/posts.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+// openboard
+//
+// user semantics:
+//
+//  Add{T}(s)    (Add{T}(s)Req)    returns {T}(s)Resp // POST
+//  Ovr{T}(s)    (Ovr{T}(s)Req)    returns {T}(s)Resp // PUT
+//  Mod{T}(s)    (Mod{T}(s)Req)    returns {T}(s)Resp // PATCH
+//  Get{T}       (Get{T}Req)       returns {T}Resp    // GET
+//  Fnd{T}s      (Fnd{T}sReq)      returns {T}sResp   // GET
+//  Rmv{T}(s)    (Rmv{T}(s)Req)    returns EmptyResp  // DELETE
+//  Unr{T}       (Unr{T}Req)       returns {T}Resp    // PATCH
+package pb;
+
+import "google/api/annotations.proto";
+import "google/protobuf/timestamp.proto";
+
+message metadata {
+	string type = 1;
+	// ?? other things ??
+}
+
+message AddPostReq {
+	string title = 1;
+	string body = 2;
+	metadata metadata = 3;
+}
+
+message PostResp {
+	uint32 id = 1;
+	string link = 2;
```

Comment:
what is this?

Reply:
After a post request is sent, this would be the response. So perhaps a post ID, a shareable link to that particular post? Other data that would then be useful?

Reply:
Cool!